pkg/apis/ceph.rook.io/v1alpha1/types.go

```diff
@@ -81,6 +81,11 @@ const (
 	ClusterStateError    ClusterState = "Error"
 )
 
+type MonSpec struct {
+	Count                int   `json:"count,omitempty"`
+	AllowMultiplePerNode *bool `json:"allowMultiplePerNode,omitempty"`
```

Comment:
does this need to be a pointer? If not a pointer, that means the default would just be false

Reply:
It had to be a pointer to distinguish between unset `nil` or set (`true` or `false`), because I chose `true` as the default.